core/trino-main/src/main/java/io/trino/execution/QueryStateMachine.java

```diff
@@ -683,6 +683,7 @@ private QueryStats getQueryStats(Optional<StageInfo> rootStage, List<StageInfo>
                 queryStateTimer.getAnalysisTime(),
                 queryStateTimer.getPlanningTime(),
                 queryStateTimer.getFinishingTime(),
+                queryStateTimer.getPlanningCpuTime(),
```

Comment:
could you move it immediately below `getPlanningTime()` (here and in other places)?

Reply:
Sure